drivers/modem/modem_sms.h

```diff
@@ -0,0 +1,47 @@
+/** @file
+ * @brief Modem SMS for SMS common structure.
+ *
+ * Modem SMS handling for modem driver.
+ */
+
+/*
+ * Copyright (c) 2021 T-Mobile USA, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
```

Comment:
What is the purpose of this pragma, why do we need it?

Reply:
From [https://en.wikipedia.org/wiki/Pragma_once](https://en.wikipedia.org/wiki/Pragma_once), "In the C and C++ programming languages, pragma once is a non-standard but widely supported preprocessor directive designed to cause the current source file to be included only once in a single compilation". Basically, it is used instead of the three lines: #ifndef and #define at the top, and #endif at the bottom.

Reply:
Please, do not do that. Just put ifndef+define+endif like the other include files in the system.